app/javascript/packages/document-capture/components/acuant-capture-canvas.jsx

```diff
@@ -260,16 +368,21 @@ function AcuantCaptureCanvas({
             transform: 'translate(-50%, -50%)',
           }}
         >
-          <h2 id={`acuant-sdk-heading-${instanceId}`}>
+          <h2 key="label" id={`acuant-sdk-heading-${instanceId}`}>
             {t('doc_auth.accessible_labels.camera_video_capture_label')}
           </h2>
-          <p id={`acuant-sdk-instructions-${instanceId}`}>
-            {t('doc_auth.accessible_labels.camera_video_capture_instructions')}
-          </p>
-          <button type="button" aria-disabled={captureType !== 'TAP'}>
+          {captureType !== 'TAP' && (
+            <p key="description" id={`acuant-sdk-instructions-${instanceId}`}>
+              {t('doc_auth.accessible_labels.camera_video_capture_instructions')}
+            </p>
+          )}
+          <button key="button" type="button" aria-disabled={captureType !== 'TAP'}>
             {t('doc_auth.buttons.take_picture')}
           </button>
         </canvas>
+        <div role="status" aria-live="polite" className="usa-sr-only">
```

Comment:
are these changes important or time sensitive enough to warrant `aria-live="assertive"`?

Reply:
> are these changes important or time sensitive enough to warrant `aria-live="assertive"`?

I think it's an important question, and I personally went back-and-forth on whether it should. Given how often this text changes, I was worried that it would be very disruptive to interrupt other announcements, especially contextual information about the dialog, or the label / description for the canvas "we will automatically take the photo". I also saw it being a source of annoyance when trying to navigate away from the dialog by clicking the "close" button that the statues would continue to be announced. I even went down a path of considering to only announce the statuses (assertively) while focus is in the canvas, but this seemed like it might be confusingly complex.

Granted, with "polite", there is the possibility the text will be delayed, which could be problematic given the timely nature of aligning the document in view. In my testing, this didn't turn out to be too much of a problem, or at least not enough that it outweighed the potential problems with announcing the texts assertively.

Reply:
> In my testing, this didn't turn out to be too much of a problem, or at least not enough that it outweighed the potential problems with announcing the texts assertively.

If it was good enough in testing then seems good to leave as-is for now at least 👍